platform/platform-app/pom.xml

```diff
@@ -359,8 +359,6 @@
         <netty41-version>4.1.9.Final</netty41-version>
         <protobuf-version>3.1.0</protobuf-version>
         <protobuf-guava-version>18.0</protobuf-guava-version>
-        <qpid-proton-j-bundle-version>0.19.0</qpid-proton-j-bundle-version>
-        <qpid-jms-client-version>0.23.0</qpid-jms-client-version>
```

Comment:
I would keep these 2 version properties names, but make them
```xml
<qpid-proton-j-bundle-version>${proton.version}</qpid-proton-j-bundle-version>
<qpid-jms-client-version>${qpid-jms.version}</qpid-jms-client-version>
```
And then move the `-The versions for proton-j and qpid-jms-client need to be kept in sync with the versions used in "artemis-amqp"` bullet in `platform/platform-app/src/main/resources/features.xml` to be a bullet in this sections's `DDF Changes` instead. That's the pattern that we used for other cases where we need to version to match the version used elsewhere in DDF.